### ISSUE TITLE
kill: Broken support for querying multiple signal names

### ISSUE THREAD

**Issue by BenWiederhake** [uutils/coreutils#6202]
Example:

```console
$ ../gnu/src/kill --version | head -n1
kill (GNU coreutils) 9.5
$ ../gnu/src/kill -l SIGUSR1 SIGUSR2
10
12
$ cargo run --all-features kill -l SIGUSR1 SIGUSR2
10
$
```

Note that the bash built-in kill also supports querying multiple names at the same time, but the Debian default binary (`procps-ng 4.0.4`) does *not*.

### More issues found while checking #6201

- Signal names in all cases, i.e. SIGUSR1 vs. sIgUsR1: Moved to #6217
- Missing `RTMAX`/`RTMIN` on Linux: Moved to #6218
- Table (`-L`) format is wrong: Moved to #6219

**Comment by m-haisham** [comment]
Noticed that the gnu starts with 1 while uutils starts with 0.

```
--- STDOUT_GNU	
- 1 HUP      Hangup
+++ STDOUT_UUTILS
+ 0 EXIT
```

I'll change uutils to start with 1 as well if that's acceptable.

**Comment by m-haisham** [comment]
I'll post a pr for third and a partial fix for first after #6201 has been merged.

The signal descriptions does not seem to be available for use in uucore atm. So probably best to add them in another pull request.

**Comment by tertsdiepraam** [comment]
@BenWiederhake, what's the status here. Can we close this with #6216 ?

**Comment by BenWiederhake** [comment]
Yeah, you're right, it should be separate issues. I was a bit too lazy when I created this issue :)
